e2e/healthcheck_test.go

```diff
@@ -4,23 +4,28 @@ import (
 	"net/http"
 	"os"
 	"os/exec"
-	"time"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
 	"github.com/onsi/gomega/gexec"
 )
 
+const (
+	// kFlag is a keyword to check whether the healthcheck is ready (or process is died)
+	kFlag = "kflag"
+)
+
 var _ = Describe("healthz check should pass", func() {
 	Context("start server with health check", func() {
 		It("should work properly", func() {
 			address, ok := os.LookupEnv("kepler_address")
 			if !ok {
 				address = "localhost:8888"
 				cmd := exec.Command(keplerBin)
-				keplerSession, err = gexec.Start(cmd, GinkgoWriter, GinkgoWriter)
+				keplerSession, err = gexec.Start(cmd, nil, nil)
 				Expect(err).NotTo(HaveOccurred())
-				time.Sleep(5 * time.Second) // wait for server start up
+				Eventually(keplerSession.Out, 30, 5).Should(gbytes.Say(kFlag))
```

Comment:
I think we should write a comment here explaining why we need this
```golang
// verifying if kepler is still alive
```

Do we need "Eventually"? 
Or we can just sleep 30s and check 
```golang
Expect(KeplerSession.Out).ShouldNot(gbytes.Say(finalizerKeyWord))
```

Reply:
This is to speed up the other case. 
Otherwise all cases have to to wait for 30s without necessity.
If we use this periodic check, the other case can finish by just 5s.

Reply:
So, there are two logics here

1. verify if the server started
2. verify if the server stopped running

In this case, we should have different strings